app/controllers/campaigns_controller.rb

```diff
@@ -1,6 +1,10 @@
 class CampaignsController < ApplicationController
   before_action :logged_in_user, only: [:new, :create]
 
+  def index
+    @campaigns = Campaign.filtered_campaigns params[:q], params[:page]
```

Comment:
params[:q] là gì e ?

Reply:
Là params q trên query url em truyền ở form_with để thực hiện filter cho campaigns ở trang xem toàn bộ chiến dịch nha anh

Reply:
ý anh là em đặt tên params sao cho nhìn cái biết nó để làm gì luôn ý em, search chẳng hạn